addons/base_vat/models/res_partner.py

```diff
@@ -14,6 +14,7 @@
                     "Install it to support more countries, for example with `easy_install vatnumber`.")
     vatnumber = None
 
+import stdnum
```

Comment:
It's not in the requirements, so you can't assume it is installed.

```python
try:
    import stdnum
except ImportError:
    stdnum = None
```

Reply:
@nim-odoo vatnumber library is on requirements and vatnumber is depend on it.
so assume it is installed.

Reply:
Indeed, but we clearly check if `vatnumber` is installed 3 lines above. So if somebody didn't install it, it won't crash. Please keep the same idea in mind.

Reply:
@nim-odoo @jva-odoo If it is not the case, then we should be consistent in base_vat/models/res_partner.py

Reply:
And somehow add it to the requirements in master, because stdnum is widely used.

Reply:
@nim-odoo this vatnumber in try and except from odoo V7.0 and vatnumber is in requirements sines V8.0 so what you see above 3 line code that needs to be improved.
So I think we can use it directly now.
still, do you think load stdnum in try and except?

Reply:
you can check at https://github.com/odoo/odoo/commit/ddd7f52a3c762ce805cf01ec78de7e265062a323

Reply:
And vatnumber depends on stdnum

Reply:
It's always better to stay on the safest side for stable branch :smile: